gobblin-service/src/main/java/org/apache/gobblin/service/modules/flowgraph/datanodes/hive/HiveDataNode.java

```diff
@@ -17,30 +17,33 @@
 
 package org.apache.gobblin.service.modules.flowgraph.datanodes.hive;
 
+
 import java.io.IOException;
 import java.net.URI;
 
 import com.google.common.base.Preconditions;
 import com.typesafe.config.Config;
 
+import java.util.Arrays;
 import joptsimple.internal.Strings;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
 
 import org.apache.gobblin.annotation.Alpha;
 import org.apache.gobblin.service.modules.flowgraph.FlowGraphConfigurationKeys;
-import org.apache.gobblin.service.modules.flowgraph.datanodes.fs.HdfsDataNode;
+import org.apache.gobblin.service.modules.flowgraph.datanodes.fs.FileSystemDataNode;
 import org.apache.gobblin.util.ConfigUtils;
 
 
 /**
- * An abstract {@link HiveDataNode} implementation. In addition to the required properties of a {@link HdfsDataNode}, an {@link HiveDataNode}
+ * An abstract {@link HiveDataNode} implementation. In addition to the required properties of a {@link FileSystemDataNode}, an {@link HiveDataNode}
```

Comment:
Drop "abstract" from the comment. It is a little misleading.

Reply:
fixed